src/explore-education-statistics-common/src/modules/table-tool/utils/__tests__/mapTableHeadersConfig.test.ts

```diff
@@ -331,6 +346,76 @@ describe('mapTableHeadersConfig', () => {
     ]);
   });
 
+  test('correctly maps term time periods to only return those in the table result', () => {
+    const testHeaderConfigWithTerms = JSON.parse(
```

Comment:
Again as an example of the object spreading here, the same can be achieved with:

```
    const testHeaderConfigWithTerms = {
        ....testHeaderConfig,
        columns: [
            {
                 value: '2017_T1',
                 type: 'TimePeriod',
            },
            ........... big long array here .........., 
        ],
     };
```